project_templates/conda_exercise/exercise_1/pear.xml

```diff
@@ -0,0 +1,165 @@
+<tool id="iuc_pear" name="Pear" version="0.9.6.1">
+    <description>Paired-End read merger</description>
+    <stdio>
+        <exit_code range="1:" />
```

Comment:
I think that in the IUC we would probably advocate the use of `<command detect_errors="exit_code">` or "aggressive"